broker/src/main/java/io/zeebe/broker/clustering/base/gossip/AtomixService.java

```diff
@@ -146,7 +125,52 @@ public Atomix get() {
     return atomix;
   }
 
-  private List<String> getRaftGroupMembers(ClusterCfg clusterCfg) {
+  private RaftPartitionGroup createRaftPartitionGroup(
+      final String rootDirectory, final String raftPartitionGroupName) {
+
+    final File raftDirectory = new File(rootDirectory, raftPartitionGroupName);
+    if (!raftDirectory.exists()) {
+      try {
+        Files.createDirectory(raftDirectory.toPath());
+      } catch (final IOException e) {
+        throw new RuntimeException("Unable to create directory " + raftDirectory, e);
+      }
+    }
+
+    final ClusterCfg clusterCfg = configuration.getCluster();
+    final DataCfg dataCfg = configuration.getData();
+    final NetworkCfg networkCfg = configuration.getNetwork();
+
+    final Builder partitionGroupBuilder =
+        RaftPartitionGroup.builder(raftPartitionGroupName)
+            .withNumPartitions(clusterCfg.getPartitionsCount())
+            .withPartitionSize(clusterCfg.getReplicationFactor())
+            .withMembers(getRaftGroupMembers(clusterCfg))
+            .withDataDirectory(raftDirectory)
+            .withFlushOnCommit();
+
+    // by default, the Atomix max entry size is 1 MB
+    final ByteValue maxMessageSize = networkCfg.getMaxMessageSize();
+    partitionGroupBuilder.withMaxEntrySize((int) maxMessageSize.toBytes());
+
+    Optional.ofNullable(dataCfg.getRaftSegmentSize())
```

Comment:
True. The user should only configure the `maxMessageSize`. The `raftSegmentSize` is basically used for testing. Similar to our `logSegmentSize`. The user should not change it. The default is 32 MB. 

I just added the just to ensure that the size is big enough.

Reply:
The user should not change it. The default is 32 MB. But he can right?

Reply:
Correct.

Reply:
I thought we want to avoid that the user is able to configure this, such that he is not able to mistakenly or because of less knowledge  misconfigure his broker

Reply:
Do you have any idea?

At least there is now a check that it doesn't break :P